esrally/telemetry.py

```diff
@@ -90,6 +92,11 @@ def detach_from_node(self, node, running):
     def on_benchmark_start(self):
         for device in self.devices:
             if self._enabled(device):
+                if self.serverless_mode and not self._available_on_serverless(device):
+                    if getattr(device, "command", None) in self.enabled_devices:
```

Comment:
I find this logic confusing. Isn't this check redundant given that `self._enabled` checks this already?

Reply:
This isn't redundant because `self._enabled` also checks if the command is internal. But I don't want to print a message for those internal commands. If we were not printing a message, we would not need this condition at all: notice that the `continue` is outside it. I've added [`57b2d5d` (#1770)](https://github.com/elastic/rally/pull/1770/commits/57b2d5d406321aef999fc520eb80b10bb37ee45e) to clarify this. What do you think?

Reply:
Ah, ofc. Thanks for clarifying. The comment helps!

Reply:
Thanks! I would not say "of course", it took me way too many attempts to get to that version. :sweat_smile: